### PR TITLE
HTBHF-586 Add cookie page content which required moving the getLanguageBase method into a common area.

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: src/web/routes/cookies.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
-const pageContent = {
+const { getLanguageBase } = require('./application/common/language')
+
+const pageContent = ({ language }) => ({
   // TODO setup cookies page to handle translations HTBHF-586
```
Comment (YetAnotherMatt) [inline src/web/routes/cookies.js]: Do we need to add a cy cookie page?

File: src/web/views/locales/en/cookies.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <th class="govuk-table__header" scope="col">Name</th>
+            <th class="govuk-table__header" scope="col">Purpose</th>
+            <th class="govuk-table__header" scope="col">Expires</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">htbhf.sid</td>
+            <td class="govuk-table__cell">Used for session management.</td>
+            <td class="govuk-table__cell">When you close your browser.</td>
```
Comment (YetAnotherMatt) [inline src/web/views/locales/en/cookies.html]: This will change, but we can update it in HTBHF-483

File: src/web/routes/cookies.js (hunk 1 above)
Comment (mikeSmith08) [reply src/web/routes/cookies.js]: Yep, that's the next sub-task. There's some work around getting the Cookie link name and translations so thought I'd keep that all together.

File: src/web/views/locales/en/cookies.html (hunk 2 above)
Comment (mikeSmith08) [reply src/web/views/locales/en/cookies.html]: Yes, for now I just copied the copy that I was given by Kate.

**Pull request merged**